org.eclipse.xpect.releng/org.eclipse.xpect.releng.utils/pom.xml

```diff
@@ -0,0 +1,10 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
miss a .settings folder in the project

Reply:
I can probably make it reusable in Xtext so in the future we can just take that on

Reply:
.settings folder is pushed.. Seemed that it was gitignore'd somewhere but add -f has done it.